frontend/src/constants/API.js

```diff
@@ -4,11 +4,8 @@ import queryString from "query-string";
 export const MINE = "/mines";
 export const MINE_LIST = "/mines";
 export const MINE_LIST_QUERY = (params) => `/mines${params}`;
+export const PARTIES_LIST_QUERY = (params = {}) => `/parties?${queryString.stringify(params)}`;
```

Comment:
I'm unfamiliar with this function type.  Does it mean:  If no param provided use '{}' as a default?

Reply:
Yep. It's just a default parameter value